README.md

```diff
@@ -49,7 +49,15 @@ vars:
     zendesk_database: your_database_name
     zendesk_schema: your_schema_name 
 ```
+### Adding Passthrough Columns
+This package includes all source columns defined in the [Zendesk Source package](https://github.com/fivetran/dbt_zendesk_source) staging models to be present within the `zendesk__ticket_enriched` and `zendesk__ticket_metrics` models. However, you may also include your own custom columns to these two end models by leveraging our passthrough variable. 
```

Comment:
I'm a little confused here. 

So far I've followed adding passthrough columns in the `source` package in the `stg_zendesk__ticket` model, and it looks like a few models use `stg_zendesk__ticket` according to the lineage graph. 

I think I may be missing something but can you explain why `zendesk__ticket_enriched` and `zendesk__ticket_metrics` are specified here? There doesn't seem to be any added variables in this modeling package to include logic to add passthrough columns.

Reply:
Sure thing! Since the `zendesk__ticket_metrics` and `zendesk__ticket_enriched` models simply bring in all the base fields from `stg_zendesk__ticket` via a select *, I wanted to highlight that the passthrough columns will be pulled through. So, I wanted to add this here so customers know they can bring the passthrough columns through to the final models.